htdocs/AntiXSS.php

```diff
@@ -1,18 +1,10 @@
 <?php
-/*
- * @file    : antixss.php
- * @created : Apr 18, 2010, 11:17:16 PM
- * @author  : Canberk BOLAT <canberk.bolat at gmail.com>
- * @version : v1.2 (Beta)
- * @license : GNU Public License v2.0 http://www.gnu.org/licenses/gpl-2.0.html
- * @desc    : PHP Anti-XSS Library.
- */
```

Comment:
@himanimukne Is there a reason you want to remove the license here? I feel like we should leave it in - if it's this person's code and they licensed in as GPLv2, we _need_ to.